application_handlers_test.go

```diff
@@ -618,6 +658,30 @@ func TestApplicationEdit(t *testing.T) {
 		t.Errorf("Expected: %v Obtained: %v", emailNotificationInfo, notificationProducer.EmailNotificationInfo)
 	}
 
+	application, _ = applicationDao.GetById(&applicationID)
+	if application.AvailabilityStatus == previousApplicationStatus {
+		t.Errorf("Invalid email notification data:")
+	}
+
+	src, err = sourceDao.GetById(&application.SourceID)
+	if err != nil {
+		t.Error(err)
+	}
+
+	if src.AvailabilityStatus == previousSourceStatus {
+		t.Errorf("Invalid email notification data:")
```

Comment:
I think you forgot the colons, ":", from the error messages

Reply:
@lpichler If you fix this message I'll merge this.

Reply:
@lindgrenj6 message was fixed and also I changed words.